clang/test/Driver/sycl-nvptx-sqrt.cpp

```diff
@@ -0,0 +1,20 @@
+// REQUIRES: clang-driver
+// REQUIRES: nvptx-registered-target
+// REQUIRES: !system-windows
```

Comment:
Why exactly is Windows unsupported here?

Reply:
I think that one might work on Windows actually, I started with the amdgcn test and was originally looking at this test:

* https://github.com/intel/llvm/blob/sycl/clang/test/Driver/rocm-device-libs.cl#L3

ROCm isn't supported on Windows so that may be why, although I feel like the Driver test may still work.

And I don't have a Windows system on hand, but I can check that next week, it should be fine to remove that for the `nvptx` test at least.

Reply:
Thanks for the explanation! Depending on whether the PR is urgent to get in, +1 for re-enabling the test for CUDA.

Reply:
Tested the `nvptx` test on Windows and confirmed it worked so I removed the restriction, left it in for the `amdgcn` test as ROCm isn't supported on Windows anyway.